tests/test_censoring.py

```diff
@@ -131,7 +132,10 @@ def test_fail_multiple_clip_single_base():
     cens_rv2 = at.clip(base_rv, -1, 1)
     cens_rv2.name = "cens2"
 
-    with pytest.raises(RuntimeError, match=r"could not be derived: {cens\d}"):
```

Comment:
This is not different from the following:

```python
base_rv = at.random.normal(0, 1)
exp_rv1 = at.exp(base_rv)
exp_rv2 = at.exp(base_rv)

conditional_logprob(exp_rv1, exp_rv2)
```

This code implies that the value of `base_rv` to two different values at the same time, which is impossible.

Reply:
I've added an error for when multiple values are bound to the same measurable term.